cmd/ormb-storage-initializer/cmd/pull-and-export.go

```diff
@@ -81,13 +83,37 @@ var pullExportCmd = &cobra.Command{
 			return err
 		}
 
+		// For model serving, need relayout option.
+		if !reLayoutOpt {
+			return nil
+		}
+
+		if err := relayoutModel(dstDir); err != nil {
+			return err
+		}
+
 		return nil
 	},
 }
 
+func relayoutModel(modelDir string) error {
+	originalDir, err := filepath.Abs(
+		filepath.Join(modelDir, consts.ORMBModelDirectory))
+	if err != nil {
+		return err
+	}
+
+	if err := os.Rename(originalDir, path.Join(modelDir, "1")); err != nil {
```

Comment:
Please add some comments about why we need it.

Reply:
comments are address, PTAL